pkg/api/avatar/avatar.go

```diff
@@ -95,7 +97,7 @@ func (a *CacheServer) Handler(ctx *models.ReqContext) {
 	if avatar.Expired() {
 		// The cache item is either expired or newly created, update it from the server
 		if err := avatar.Update(); err != nil {
-			log.Debug("avatar update", "err", err)
+			alog.Debug("avatar update", "err", err)
```

Comment:
I wonder if we could use a contextual logger instead? It seems like a request-scoped log message.